tool/teleport/common/teleport.go

```diff
@@ -299,7 +299,7 @@ func Run(options Options) (app *kingpin.Application, executedCommand string, con
 	dbConfigureCreate.Flag("gcp-instance-id", "(Only for Cloud SQL) GCP Cloud SQL instance identifier.").StringVar(&dbConfigCreateFlags.DatabaseGCPInstanceID)
 	dbConfigureCreate.Flag("ca-cert-file", "Database CA certificate path.").StringVar(&dbConfigCreateFlags.DatabaseCACertFile)
 	dbConfigureCreate.Flag("output",
-		"Write to stdout with -o=stdout, default config file with -o=file or custom path with -o=file:///path").Short('o').Default(
+		`Write to stdout with "-o stdout", default config file with "-o file" or custom path with --output=file:///path`).Short('o').Default(
```

Comment:
I would use the long `--output` option consistently in the message. Switching between `-o` and `--output` is awkward.

Reply:
thanks @zmb3, update to all `--output`